base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {ebc}{c}   {ebc}  {}
+\DeclareFontSeriesChangeRule {ebc}{sc}  {ebsc} {}
+\DeclareFontSeriesChangeRule {ebc}{sx}  {ebsx} {}
+\DeclareFontSeriesChangeRule {ebc}{x}   {ebx}  {}
+\DeclareFontSeriesChangeRule {ebc}{ex}  {ebex} {}
+\DeclareFontSeriesChangeRule {ebc}{ux}  {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsc}{ul} {ulsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{el} {elsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{l}  {lsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sl} {slsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sb} {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{b}  {bsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{eb} {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{ub} {ubsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebsc}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebsc}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsc}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsc}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsc}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {eb}{uc}   {ebuc} {}
+\DeclareFontSeriesChangeRule {eb}{ec}   {ebec} {}
+\DeclareFontSeriesChangeRule {eb}{c}    {ebc}  {}
+\DeclareFontSeriesChangeRule {eb}{sc}   {ebsc} {}
+\DeclareFontSeriesChangeRule {eb}{sx}   {ebsx} {}
+\DeclareFontSeriesChangeRule {eb}{x}    {ebx}  {}
+\DeclareFontSeriesChangeRule {eb}{ex}   {ebex} {}
+\DeclareFontSeriesChangeRule {eb}{ux}   {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsx}{ul} {ulsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{el} {elsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{l}  {lsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sl} {slsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{sb} {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{b}  {bsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{eb} {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{ub} {ubsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebsx}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebsx}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsx}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsx}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebx}{ul}  {ulx}  {}
+\DeclareFontSeriesChangeRule {ebx}{el}  {elx}  {}
+\DeclareFontSeriesChangeRule {ebx}{l}   {lx}   {}
+\DeclareFontSeriesChangeRule {ebx}{sl}  {slx}  {}
+\DeclareFontSeriesChangeRule {ebx}{sb}  {sbx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{b}   {bx}   {}
+\DeclareFontSeriesChangeRule {ebx}{eb}  {ebx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{ub}  {ubx}  {bx}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebx}{ub}  {ubx}  {ebx}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.